### PR TITLE
Fix for BSC#1218070.

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description
Added using the modprobe command to mount a file system.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP6/openSUSE Leap 15.6
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2

- SLE 12
  - [ ] SLE 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/storage_filesystems.xml [hunk 1]
```diff
@@ -2213,13 +2184,55 @@ The file system has the following characteristics:
...
+<variablelist>
+  <varlistentry>
+    <term><keycap>y</keycap>es</term>
+    <listitem>
+      <para>
+        The module will be loaded and removed from the blacklist. Therefore, the module will be
+        loaded automatically in future without any other prompt.
+      </para>
+      <para>
+        The onfiguration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
```
Comment (mwilck) [inline xml/storage_filesystems.xml:2199]:
```suggestion
        The configuration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
```

File: xml/storage_filesystems.xml [hunk 2]
```diff
@@ -2213,13 +2184,55 @@ The file system has the following characteristics:
...
+</varlistentry>
+<varlistentry>
+  <term><keycap>e</keycap>ver</term>
+  <listitem>
+    <para>
+      The module will be loaded, but autoloading is disabled even for future and the prompt will
+      not be displayed anymore.
+    </para>
+    <para>
+      The onfiguration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
```
Comment (mwilck) [inline xml/storage_filesystems.xml:2222]:
```suggestion
      The configuration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
```

File: xml/storage_filesystems.xml [hunk 3]
```diff
@@ -2213,13 +2184,55 @@ The file system has the following characteristics:
...
+      created. Remove the file to undo the changes you performed.
+    </para>
+  </listitem>
+</varlistentry>
+<varlistentry>
+  <term><keycombo><keycap function="control"/><keycap>c</keycap></keycombo></term>
+  <listitem>
+    <para>
+      To interrup the module loading.
```
Comment (mwilck) [inline xml/storage_filesystems.xml:2231]:
```suggestion
      To interrupt the module loading.
```

Comment (mwilck) [reply xml/storage_filesystems.xml:2231]: Not sure if the possibility of typing `ctrl-c` needs to be mentioned explicitly here.

**Review by mwilck — CHANGES_REQUESTED** [review]
Looks good, I made some minor annotations.

File: xml/storage_filesystems.xml [hunk 4]
```diff
@@ -2213,13 +2184,55 @@ The file system has the following characteristics:
...
+      The configuration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
+      created. Remove the file to undo the changes you performed.
+    </para>
+  </listitem>
+</varlistentry>
+<varlistentry>
+  <term><keycombo><keycap function="control"/><keycap>c</keycap></keycombo></term>
+  <listitem>
+    <para>
+      To interrupt the module loading.
```
Comment (dariavladykina) [inline xml/storage_filesystems.xml:2231]:
```suggestion
      This option is used to interrupt the module loading.
```

File: xml/storage_filesystems.xml [hunk 5]
```diff
@@ -2213,13 +2184,55 @@ The file system has the following characteristics:
...
+      you will see the prompt above again.
+    </para>
+  </listitem>
+</varlistentry>
+<varlistentry>
+  <term><keycap>e</keycap>ver</term>
+  <listitem>
+    <para>
+      The module will be loaded, but autoloading is disabled even for future and the prompt will
+      not be displayed anymore.
```
Comment (dariavladykina) [inline xml/storage_filesystems.xml:2219]:
```suggestion
      The module will be loaded, but autoloading is disabled even for future use, and the prompt 
      will no longer be displayed."
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see my suggestions. Thanks!

**Pull request merged**